README.md

```diff
@@ -893,6 +895,8 @@ Buffer-local mappings also take a buffer number as their first argument (`0` set
 ```lua
 vim.api.nvim_set_keymap('n', '<leader><Space>', ':set hlsearch!<CR>', { noremap = true, silent = true })
 -- :nnoremap <silent> <leader><Space> :set hlsearch<CR>
+vim.api.nvim_set_keymap('n', '<leader>tegf',  [[<cmd>lua require('telescope.builtin').git_files()<CR>]], { noremap = true, silent = true })
+-- :nnoremap <silent> <leader>tegf :lua require('telescope.builtin').git_files()<CR>
```

Comment:
I think the Vimscript version should be `<Cmd>lua require...`
Also, small nitpick but I prefer to stick to the convention of the Neovim documentation and spell it `<Cmd>` rather than `<cmd>` :)

Reply:
Is there a linter that checks this kind of stuff automatically for neovim? Would be good to add to the guide.